frontend/src/metabase/querying/components/FilterModal/utils/search.ts

```diff
@@ -11,19 +11,29 @@ export function searchGroupItems(
   searchText: string,
 ): GroupItem[] {
   const searchValue = searchText.toLowerCase();
+  const searchValueHasSegments = t`segments`.includes(searchValue);
```

Comment:
It took me a while to figure out why the value isn't ```searchValue.includes(t`segments`)```, but it was the variable name that was confusing.

```suggestion
  const isSearchingForSegments = t`segments`.includes(searchValue);
```